frontend/server/src/Controllers/Problem.php

```diff
@@ -899,14 +899,12 @@ public static function apiUpdate(\OmegaUp\Request $r) {
     /**
      * @psalm-suppress MixedInferredReturnType Psalm cannot effectively analyze templated arrays this way
      * @psalm-suppress MismatchingDocblockReturnType Psalm cannot effectively analyze templated arrays this way
-     * @psalm-suppress MixedAssignment Psalm cannot effectively analyze templated arrays this way
      * @template T
      * @param T $array
      * @return T
      */
     private static function arrayDeepCopy($array): array {
         $copy = [];
-        /** @var string $key */
         foreach ($array as $key => $value) {
```

Comment:
```suggestion
        /**
         * @var string $key
         * @var mixed $value
         */
        foreach ($array as $key => $value) {
```

aquí está bien decir que `$value` es mixto. esto debería eliminar un `MixedAssignment` y los dos `MixedArrayOffset`. el último `MixedAssignment` se debería poder ir si le agregas `/** @var mixed $value */` arriba de L912.

Reply:
Listo. Ya quedaron.